lightning/src/ln/channelmanager.rs

```diff
@@ -14835,6 +14835,22 @@ where
 						if payment_claim.mpp_parts.is_empty() {
 							return Err(DecodeError::InvalidValue);
 						}
+						{
```

Comment:
Sentence is phrased weirdly in the commit message: "However, on startup, we check that failing to move a payment from `claimable_payments` to `claiming_payments` we check that it is not present in `claiming_payments`."